src/web/components/App.tsx

```diff
@@ -326,6 +332,12 @@ export const App = ({ CAPI, NAV }: Props) => {
 		});
 	}, []);
 
+	useOnce(() => {
+		setBrazeMessages(
+			buildBrazeMessages(isSignedIn as boolean, CAPI.config.idApiUrl),
+		);
+	}, [isSignedIn, CAPI.config.idApiUrl]);
```

Comment:
I think you're already aware but just to be sure this array are the things that must be defined before the body executes. So having `CAPI.config.idApiUrl` is a guard against that not being defined (which I presume is unlikely but good to have) and then `isSignedIn` is what is really being used here